sql-statements/sql-statement-create-user.md

```diff
@@ -95,6 +97,24 @@ CREATE USER 'newuser5'@'%' ACCOUNT LOCK;
 Query OK, 1 row affected (0.02 sec)
 ```
 
+创建一个带注释的用户。
+```sql
+CREATE USER 'newuser6'@'%' COMMENT 'This user is created only for test';
+```
+
+```
+Query OK, 1 row affected (0.02 sec)
+```
```

Comment:
````suggestion
创建一个带注释的用户。

```sql
CREATE USER 'newuser6'@'%' COMMENT 'This user is created only for test';
SELECT * FROM user_attributes;
```

```
+-----------+------+---------------------------------------------------+
| USER      | HOST | ATTRIBUTE                                         |
+-----------+------+---------------------------------------------------+
| newuser6  | %    | {"comment": "This user is created only for test"} |
+-----------+------+---------------------------------------------------+
1 rows in set (0.00 sec)
```
````